constraints.txt

```diff
@@ -0,0 +1,2 @@
+git+git://github.com/david4096/ga4gh-client.git@protobuf31#egg=ga4gh_client
```

Comment:
Here's another solution that makes things easier for developers: keep the `requirements.txt` file as is and put into the `constraints.txt` file commented-out lines (with the egg fragment) of what would be there if a developer wanted to point to git branches instead of released packages.  The developer would only have to uncomment the line (or, at worst, it would be more obvious what to do to modify it).

I would be against adding more steps to the release process, if at all possible.  Manual steps in releases = inevitable screwups.  Indeed, the closer we can get to a one-button release (as advocated in https://www.amazon.com/Building-Scalable-Web-Sites-Applications/dp/0596102356 ), the better.